### ISSUE TITLE
Check UI against Web Content Accessibility Guidelines

### ISSUE THREAD

**Issue by joeyparrish** [google/shaka-player#3146]
We should check that the Shaka Player UI conforms to the Web Content Accessibility Guidelines (WCAG) 2.1: https://www.w3.org/TR/WCAG21/

Though we built the UI against Google-internal guidelines, it would be good to double-check everything on the WCAG checklist so that we could catch any mistakes or omissions and correct them.  Then we could confidently document the UI as conforming to that guide.

**Comment by ningl2** [comment]
Some of the tests we did revealed the following issues:

o	No titles for controls
o	Focus problems – on some cases the focus disappears, on menu, on new full screen.
o	Part of the controls are triggering on key down when it should be on key up.
o	Missing roles for some of the controls
o	Not supporting high contrast mode

**Comment by joeyparrish** [comment]
Thank you for the feedback!

Just to clarify, are these issues strictly in the player UI, or also in the broader demo app?  (We know the demo app lags behind the player UI in terms of localization and accessibility, but it wouldn't be part of your app.)

**Comment by avelad** [comment]
@KimKyuHoi Do you want to address this?

**Comment by KimKyuHoi** [comment]
> @KimKyuHoi Do you want to address this?

Yes, I'd love to! Thanks for your suggestion :)
